### PR TITLE
Remove authentication from /openapi/v2 call

### LINKED ISSUE #371: /openapi/v2 returns 401 when an authenticated call is made
If the `/openapi/v2` endpoint is called using a bearer token, then that token is checked and a 401 Unauthorized is returned if the token is invalid. This is poor behavior on K8s's part, but until then, the request to `/openapi/v2` needs to be made without a bearer token.

### PR DESCRIPTION
**Pull request by ajpauwels**
fixes #371

### CODE REVIEW

File: lib/swagger-client.js [hunk 1]
```diff
@@ -33,14 +33,14 @@ const Request = require('./request')
 
 class Root extends Component {
   _getSpec (pathname) {
-    return this.http.http({ method: 'GET', pathname })
+    const noAuthHTTP = new Request({
+      url: this.http.requestOptions.baseUrl,
+      ca: this.http.requestOptions.ca
+    })
+    return noAuthHTTP.http({ method: 'GET', pathname })
```
Comment (silasbw) [inline lib/swagger-client.js]: What if we added a `noAuth` option to `http`?  One benefit of doing that is if we modify `Request` or add a new version and a user constructs it with some custom configuration that we wouldn't know how to replicate when we construct it here.

Thoughts?

File: lib/swagger-client.js [hunk 2]
```diff
@@ -33,14 +33,14 @@ const Request = require('./request')
 
 class Root extends Component {
   _getSpec (pathname) {
-    return this.http.http({ method: 'GET', pathname })
```
Comment (silasbw) [inline lib/swagger-client.js]: I was imagining something like:

```js
return this.http.http({ method: 'GET', pathname, noAuth: true })
```

File: lib/swagger-client.js (hunk 1 above)
Comment (silasbw) [reply lib/swagger-client.js]: Thoughts on https://github.com/godaddy/kubernetes-client/pull/372/files#r242026927 ?

**Comment by ajpauwels** [comment]
A little swamped this week but will get to this over the weekend.

**Comment by ajpauwels** [comment]
@silasbw Addressed comments in latest PR

**Comment by silasbw** [comment]
Thanks again.  Published with `kubernetes-client@6.6.0`.

**Pull request merged**